### PR TITLE
Vmi/chips for scenario run status prod 8407

### PR DESCRIPTION
**Pull request by COSMO-EMI**
Dev Review rules:
- 3 PR max
- at the second feedback, prefer meet instead of github conversation

Definition of Done:
- [ ] Testing
- [ ] No error logs in web console (except iFrame errors)
- [ ] Create or Update documentation (README file for users/integrators)
- [ ] Check the [test page](https://spaceport.cosmotech.com/confluence/pages/viewpage.action?spaceKey=QA&title=Azure+Sample+Webapp) in order to manually validate the feature 
- [ ] Clean code (regroup configuration variable, typos, no console.log...)

### CODE REVIEW

File: src/theme/custom/cosmoDark/palette.js [hunk 1]
```diff
@@ -9,10 +9,12 @@ const themeColors = {
   background: '#191919',
   backgroundVariant: '#1F1F1F',
   surface: '#2E2E2E',
-  error: '#FF667F',
+  error: '#D1440C',
   warning: '#FB8C00',
-  success: '#19E152',
-  info: '#1E88E5',
+  success: '#87B84E',
+  // for light theme:
```
Comment (jreynard-code) [inline src/theme/custom/cosmoDark/palette.js:15]: You can remove the comment here and set the value in `custom/cosmoLight/palette.js`

Comment (COSMO-VMI) [reply src/theme/custom/cosmoDark/palette.js:15]: Thanks, it's much better now!

File: src/theme/custom/cosmoDark/palette.js [hunk 2]
```diff
@@ -64,6 +66,7 @@ export default {
     paper: themeColors.background,
     secondary: themeColors.backgroundVariant,
     card: themeColors.surface,
+    dialogPaper: themeColors.surface,
```
Comment (Nikeleos) [inline src/theme/custom/cosmoDark/palette.js:69]: Once again, is there any way to fusion card an dialogPaper as a unique surface type ?

**Review by Nikeleos — COMMENTED** [review]
One little remark otherwise LGTM.

File: src/theme/custom/cosmoDark/palette.js (hunk 2 above)
Comment (COSMO-VMI) [reply src/theme/custom/cosmoDark/palette.js:69]: Sharing the same color does not mean the components are the same, so I don't get what you want here.

**Pull request merged**